SharedProjects/Babylon2GLTF/GLTFExporter.Texture.cs

```diff
@@ -160,7 +160,16 @@ private GLTFTextureInfo ExportTexture(BabylonTexture babylonTexture, GLTF gltf,
                     string textureUri = name;
                     if (!string.IsNullOrWhiteSpace(exportParameters.textureFolder))
                     {
-                        textureUri = PathUtilities.GetRelativePath( exportParameters.outputPath,exportParameters.textureFolder) + "/"+ name;
+                        textureUri = PathUtilities.GetRelativePath( exportParameters.outputPath,exportParameters.textureFolder);
+                        if (textureUri == ".\\")
+                        {
+                            //texture is in same location of model
+                            textureUri = name;
+                        }
+                        else
+                        {
+                            textureUri = Path.Combine(textureUri, name);
```

Comment:
On windows, we end up having backslashes in our image URI path, after getting this relative path. should we replace that with forward slashes?